tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -155,6 +155,25 @@ def test_exponential_scheduler(max_epochs, initial_value, gamma):
     exp_state_parameter_scheduler.load_state_dict(state_dict)
 
 
+@pytest.mark.parametrize(
+    "max_epochs, initial_value, gamma", [(3, 10, 0.99)],
+)
+def test_multiple_scheduler_with_save_history(max_epochs, initial_value, gamma):
+    engine = Engine(lambda e, b: None)
+    exp_state_parameter_scheduler1 = ExpStateScheduler(
+        param_name="exp_scheduled_param1", initial_value=initial_value, gamma=gamma, save_history=True
+    )
+    exp_state_parameter_scheduler2 = ExpStateScheduler(
+        param_name="exp_scheduled_param2", initial_value=initial_value, gamma=gamma, save_history=True
+    )
+    exp_state_parameter_scheduler1.attach(engine, Events.EPOCH_COMPLETED)
+    exp_state_parameter_scheduler2.attach(engine, Events.EPOCH_COMPLETED)
+    try:
```

Comment:
Can we find another way to do the test here?

Reply:
@KickItLikeShika  thanks for your comment! , I am not fully satisfied either, will try to find another way..